### DISCUSSION TITLE
audiocast.lan

### DISCUSSION THREAD

**Discussion by jcdamechanic** [Security-Onion-Solutions/securityonion#8728] [General]
Hello, recently i've seen a random device on my network that has the host name "audiocast.lan". I've done a nmap scan on the IP only to discover a bunch of peripherals ports(9999,1007,8443,8009,8012,8008,10101,7000,10001,9000) open on it.  I have attached a screenshot of the results. If anyone could please advice I would really appreciate it. Thanks.

![Nmap audiocast lan half way](https://user-images.githubusercontent.com/55727151/189918911-7fd2af0a-49d8-4936-a5a1-c7f208144711.png)

**Comment by xfaith** [comment]
Do you have any devices plugged into say a legacy stereo?  
https://audiocast.io/  I am looking at the obvious items.
